force-app/main/default/classes/example/AccountDomainTest.cls

```diff
@@ -1,103 +1,98 @@
 @isTest
 public with sharing class AccountDomainTest {
+  
+  private static final UniversalMocker mockService;
+  private static final AccountDBService mockServiceStub;
+  private static final AccountDomain sut; // system under test
+  
+  static {
```

Comment:
@Thieveryshoe thanks for the PR, this is great. May I request that you use a static method annotated with `@TestSetup` instead of a static block for initialization? That would be more in accordance with Salesforce's guidelines

Reply:
I'm not sure this is possible.  I'm not very familiar with the `@testSetup` decorator.

You can't use the final keyword and set the values inside of the `@testSetup` method.  Hence the static block.  If `final` is dropped we end up with `variable does not exist` errors throughout the class.

Again, I'm not very familiar with the `@testSetup` decorator.  Am I missing something obvious?

Reply:
@Thieveryshoe you are right, forgot that `@TestSetup` runs in a different transaction than the test methods. Let's roll with the static block! Can you change `AccountDBService mockService = (AccountDBService) mockService.createStub();` to ```mockServiceStub = (AccountDBService) mockService.createStub(); 
sut = new AccountDomain(mockServiceStub);```  Don't see the need to create another instance of the stub

Reply:
Good catch.  Committed the updated changes.